### PR TITLE
Update Wix version

### PR DESCRIPTION
**Pull request by wtgodbe**
Part of https://github.com/dotnet/aspnetcore/issues/55087

### CODE REVIEW

File: eng/targets/Wix.Common.props [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
     <ProductVersion>3.14</ProductVersion>
-    <WixVersion>3.14.1.8722</WixVersion>
```
Comment (joeloff) [inline eng/targets/Wix.Common.props]: FYI the 3.14.1.8722 is the version used for the zip, the full version you have in this change is for the nupkg.  If you want the nupkg, then you can use the Arcade property `MicrosoftSignedWixVersion` for that

Comment (wtgodbe) [reply eng/targets/Wix.Common.props]: Thanks, updated

**Pull request merged**